### ISSUE TITLE
ctc_loss gives  No valid path found incorrectly

### ISSUE THREAD

**Issue by danFromTelAviv** [tensorflow/tensorflow#23332]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): I used stock functions as is but through keras
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): win 10 
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below): 1.8
- Python version: 3.6.3
- CUDA/cuDNN version: 7.1 to the best of my memory
- GPU model and memory: ti 1070 6bg 

I am using ctc_loss() and I get the error "T:\src\github\tensorflow\tensorflow\core\util\ctc\ctc_loss_calculator.cc:144] No valid path found." 
I am pretty sure that all the data I fed in included some valid path - that is the length of the labels is ~.5x the length of the inputs and the labels have very few repetitions. 

Its a bit hard to reproduce since I am not sure which sample is causing the issue.
I can copy paste the batch that I feed into my network if that helps?
